src/components/ChatView.tsx

```diff
@@ -621,6 +629,13 @@ const ChatView = forwardRef<ChatViewHandle, ChatViewProps>(({
                                                                 setLastCopiedContent(selection.toString());
                                                             }
                                                         }}
+                                                        onCut={() => {
+                                                            // Store the c content for same-window paste functionality
```

Comment:
in some places both onCopy and onCut is added but in some places only onCopy is added - can we make sure that both onCopy and onCut is added everywhere